requirements-test.txt

```diff
@@ -9,3 +9,4 @@ redis==3.4.1
 requests==2.23.0
 git+git://github.com/getsentry/sentry-python@e234998ae82a9cffa6fb3718801c55ba24a86bab#egg=sentry_sdk
 werkzeug==0.15.5
+sentry-relay==0.8.5
```

Comment:
Please do not include librelay in Relay binary integration tests. This forces another (release!) build of the python library, which increases test times, and unnecessarily increases the test surface. Please hard code the expectations, instead.

Reply:
looking for a solution...

Reply:
After discussing the alternatives with @untitaker we came to the conclusion that the most reasonable approach is to leave the sentry-relay dependency and add a dependabot task that will check for the dependencies on a few packages in the `requirements-test.txt` ( sentry-relay, sentry-python, pytest-sentry)